src/App.vue

```diff
@@ -1,31 +1,87 @@
 <template>
```

Comment:
@ignaHeinz did modify the same file, decide who will modify it in the pull requests